egs/swbd/s5c/local/reverberate/IR_stats.m

```diff
@@ -0,0 +1,348 @@
+function [rt,drr,cte,cfs,edt] = IR_stats(x, fs, varargin)
+%function [rt,drr,cte,cfs,edt] = IR_stats(filename, varargin)
```

Comment:
Hmm.. in that case we will just have to rely on the openslr.org for the
RIRs and noise. The scripts which prepare these simulated RIRs for upload
to openslr.org, will not be checked in.

On Wed, Apr 6, 2016 at 10:52 PM, Daniel Povey <notifications@github.com>
wrote:

> In egs/swbd/s5c/local/reverberate/IR_stats.m
> <https://github.com/kaldi-asr/kaldi/pull/667#discussion_r58812863>:
>
> > @@ -0,0 +1,348 @@
> > +function [rt,drr,cte,cfs,edt] = IR_stats(x, fs, varargin)
> > +%function [rt,drr,cte,cfs,edt] = IR_stats(filename, varargin)
>
> I don't see any point distributing scripts that have a dependency on code
> that we can't distribute. This would only work on our grid -> pointless to
> check in, surely.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/667/files/1683e258c9f571271668ab6e1d2b0898e7b7ed33#r58812863>
>